library/core/functions.general.php

```diff
@@ -3147,12 +3147,6 @@ function removeKeysFromNestedArray($Array, $Matches) {
     }
 }
 
-if (!function_exists('removeQuoteSlashes')) {
```

Comment:
You don't want to remove this function at this point. Just make a call to `deprecated()`.